pkg/app/desired_state_file_loader.go

```diff
@@ -36,6 +36,8 @@ type desiredStateLoader struct {
 	remote      *remote.Remote
 	logger      *zap.SugaredLogger
 	valsRuntime vals.Evaluator
+
+	lockFile string
```

Comment:
I don't see the assignment of this field. Are you sure it is useful? @lassizci

Reply:
There seems to be existing convention to have the fields in the [desiredStateLoader](https://github.com/helmfile/helmfile/blob/44bf5590d9a574c7bdf95272cdebef2ee9e25a35/pkg/app/desired_state_file_loader.go#L24-L41) regardless of whether they are assigned or not.


this: https://github.com/helmfile/helmfile/blob/44bf5590d9a574c7bdf95272cdebef2ee9e25a35/pkg/app/desired_state_file_loader.go#L168

cloud also be initialized by just using empty string, such as:
```go
c := state.NewCreator(a.logger, a.fs, a.valsRuntime, a.getHelm, a.overrideHelmBinary, a.remote, a.enableLiveOutput, "")
```
but then it wouldn't follow the existing style and I think `""` is less obvious, which argument are we passing.

Reply:
@mumoshu WDYT?